core/src/components/slides/slides.tsx

```diff
@@ -521,6 +521,9 @@ export class Slides implements ComponentInterface {
         <div class="swiper-wrapper">
           <slot></slot>
         </div>
+        
+        <slot name="container"></slot>
```

Comment:
Would it be useful to have a slot that is rendered before `.swiper-wrapper`? In that case, it may be useful to have the slots named `top` and `bottom`.

Reply:
Sure! I did that - added slot top and bottom, but maybe (just maybe), better naming would be "before" and "after", which means "before/after" slides? If you think that top/bottom is ok then I'm fine with that.

Reply:
"before" and "after" could work too. I'm not 100% sold on the "top" and "bottom" naming. Maybe @brandyscarney has ideas?

Reply:
@brandyscarney Any comment regarding naming the slots?

Reply:
@brandyscarney Knock knock ;-)